apps/protocol-frontend/src/components/ProfileForm.tsx

```diff
@@ -13,14 +13,20 @@ import useDisplayName from '../hooks/useDisplayName';
 import useLinearUserDisconnect from '../hooks/useLinearUserDisconnect';
 import useUserCustomUpdate from '../hooks/useUserCustomUpdate';
 import ProfileDaos from './ProfileDaos';
+import { CgLinear, SiDiscord } from 'react-icons/all';
+import useDiscordUserDisconnect from '../hooks/useDiscordUserDisconnect';
 
 const LINEAR_CLIENT_ID = import.meta.env.VITE_LINEAR_CLIENT_ID;
 const LINEAR_REDIRECT_URI = import.meta.env.VITE_LINEAR_REDIRECT_URI;
 const BACKEND_ADDR = `${BASE_URL}`;
 
+const DISCORD_CLIENT_ID = '1062060045929549844';
+const REDIRECT_URL = 'http://localhost:4000/discord/oauth';
```

Comment:
These should be environment variables so we can use this in multiple environments.

Reply:
Great call -- can then reuse these env vars in the Signature flow standalone page 🔥